tests/fast-integration/kcp/client.js

```diff
@@ -101,6 +101,18 @@ class KCPWrapper {
     return JSON.parse(result);
   }
 
+  async reconciliations(query) {
+    let args = ['reconciliations', `${query.parameter}`, '--output', 'json'];
+    if (query.shootName) {
+      args = args.concat('--shoot', `${query.shootName}`);
+    }
+    if (query.shootName) {
+      args = args.concat('--scheduling-id', `${query.shootName}`);
```

Comment:
shouldn't it be something like:

```suggestion
      args = args.concat('--scheduling-id', `${query.schedulingID}`);
```